allure-generator/src/main/java/io/qameta/allure/allure1/Allure1Plugin.java

```diff
@@ -71,19 +72,33 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(Allure1Plugin.class);
     private static final String UNKNOWN = "unknown";
     private static final String MD_5 = "md5";
+    private static final String ISSUE_URL_PROPERTY = "allure.issues.tracker.pattern";
+    private static final String TMS_LINK_PROPERTY = "allure.tests.management.pattern";
 
     private final ObjectMapper mapper = new ObjectMapper();
 
     @Override
     public void readResults(final Configuration configuration,
                             final ResultsVisitor visitor,
                             final Path resultsDirectory) {
+        loadAllureProperties(resultsDirectory);
         final RandomUidContext context = configuration.requireContext(RandomUidContext.class);
         getStreamOfAllure1Results(resultsDirectory).forEach(testSuite -> testSuite.getTestCases()
                 .forEach(testCase -> convert(context.getValue(), resultsDirectory, visitor, testSuite, testCase))
         );
     }
 
+    private void loadAllureProperties(final Path resultsDirectory) {
+        final Path propertiesFile = resultsDirectory.resolve("allure.properties");
+        if (Files.exists(propertiesFile)) {
+            try (final FileInputStream propFile = new FileInputStream(propertiesFile.toFile())) {
+                System.getProperties().load(propFile);
```

Comment:
The problem is that system properties should override properties from file. You can create load properties from file to separate object, and then pass it to methods